runtime/lua/vim/lsp.lua

```diff
@@ -132,9 +132,10 @@ local format_line_ending = {
   ['mac'] = '\r',
 }
 
+---@private
 ---@param bufnr (number)
 ---@return string
-local function buf_get_line_ending(bufnr)
+function lsp._buf_get_line_ending(bufnr)
```

Comment:
This and `buf_get_full_text` probably makes more sense to have in `lsp.util`?

Reply:
No, `lsp.util` *delendam est*. Can't we find a better, permanent, place for it?

Reply:
`lsp._util`?